scala-package/native/src/main/native/ml_dmlc_mxnet_native_c_api.cc

```diff
@@ -1711,3 +1714,435 @@ JNIEXPORT jint JNICALL Java_ml_dmlc_mxnet_LibInfo_mxRtcFree
   int ret = MXRtcFree(handle);
   return ret;
 }
+
+JNIEnv *globalEnv;
+std::unordered_map<std::string, jobject> globalOpPropMap;
+std::unordered_map<std::string, jobject> globalOpMap;
+
+JNIEXPORT jint JNICALL Java_ml_dmlc_mxnet_LibInfo_mxCustomOpRegister
+  (JNIEnv *env, jobject obj, jstring jregName, jobject jopProp) {
+
+  globalEnv = env;
```

Comment:
why I use global variable here is because, if you want to use a local variable inside a c++ lambda,
this variable needs to be captured, then you can not convert this lambda to function pointer.